### PR TITLE
Add 22 Domain of CV DOIT PERSADA INDONESIA

### PR DESCRIPTION
**Pull request by doitindonesia**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====
doIT Persada Indonesia is a company engaged in the field of IT services for cloud hosting providers, website creation and management, digital marketing solutions.

Organization Website: 
====
Provide the website address of 
https://doit.co.id

Reason for PSL Inclusion
====
As our cloud hosting service, website and managed service, and digital marketing service, customers can either a) use their own domains, or b) choose one of the domains we own, and issue a subdomain to obtain the dedicated domain without managing their own domain. In the latter case, each subdomain on the shared domain belongs to a different customer. It is necessary for our domains to be added to the list, in order to prevent multi-site cookies.

DNS Verification via dig
=======

```
dig +short TXT _psl.azkasyah.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.ciruasland.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.doit.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.dopay.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.margalingkarjakarta.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.mikatama.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.petrolin.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.rehlata.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.specta.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.tmacontractor.co.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.mixstore.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.nobar.web.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.domail.biz.id
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.acisindonesia.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.akrilikdisplay.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.akrilikexpress.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.auliadigitalprint.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.duniaacrylic.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.dutaled.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.icebergwindowfilms.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.paketsolusi.com
https://github.com/publicsuffix/list/pull/1707
dig +short TXT _psl.royalbravosolusindo.com
https://github.com/publicsuffix/list/pull/1707
```
-->

Results of Syntax Checker (`make test`)
=========
```
cd linter;                                \
  ./pslint_selftest.sh;                     \
  ./pslint.py ../public_suffix_list.dat;
test_NFKC: OK
test_allowedchars: OK
test_dots: OK
test_duplicate: OK
test_exception: OK
test_punycode: OK
test_section1: OK
test_section2: OK
test_section3: OK
test_section4: OK
test_spaces: OK
test_wildcard: OK
test -d libpsl || git clone --depth=1 https://github.com/rockdaboot/libpsl;   \
  cd libpsl;                                                                    \
  git pull;                                                                     \
  echo "EXTRA_DIST =" >  gtk-doc.make;                                          \
  echo "CLEANFILES =" >> gtk-doc.make;                                          \
  autoreconf --install --force --symlink;
Already up to date.
Can't exec "autopoint": No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm line 293.
autoreconf: error: autopoint failed with exit status: 2
make: *** [Makefile:18: libpsl-config] Error 2
```

### CODE REVIEW

**Comment by doitcoid** [comment]
Hi, We just edited sorting our domain, please check.

**Comment by doitcoid** [comment]
Hi, We just edited sorting, please check. Thank you.

**Review by doitindonesia — COMMENTED** [review]
We just edited Sorting. Thank you

**Review by dnsguru — CHANGES_REQUESTED** [review]
NO, THIS DOES NOT GO IN ICANN SECTION

MOVE THESE TO PRIVATE SECTION, Sorted by the organization name, then these need to be sorted by TLD, then SLD, then 3LD within the Pull Request.  

PLEASE READ GUIDELINES to see if there is more you need to do in making a conforming Pull Request.   

Once this is done, use patience, as these wasted cycles and back and forth as well as the very long queue of Pull Requests ahead of this request - all work against the time available from volunteer maintainers.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
+2// This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (dnsguru) [inline public_suffix_list.dat:1]: This would get immediately rejected, remove this 2

File: public_suffix_list.dat [hunk 2]
```diff
@@ -867,6 +867,31 @@ int.cv
 nome.cv
 org.cv
 
+// CV DOIT PERSADA INDONESIA : https://doit.co.id
```
Comment (dnsguru) [inline public_suffix_list.dat:870]: Andhy please move the whole submitted section out of the ICANN section and down into the #PRIVATE section ordered by where your organization's name would be the next alphabetical org based upon other entries' commented organization name.

**Review by doitindonesia — COMMENTED** [review]
We just edited, thank you.

File: public_suffix_list.dat (hunk 1 above)
Comment (doitcoid) [reply public_suffix_list.dat]: I just edited, thank you

File: public_suffix_list.dat (hunk 2 above)
Comment (doitcoid) [reply public_suffix_list.dat]: I just edited too, thank you

**Comment by dnsguru** [comment]
Almost....  and these iterations are not something we are resourced for.

you moved section but need to alphabetically order the sorting of the
entries
.com
.biz.id
.co.id
Etc

|Entries Sorted by the organization name, 《DONE》
|then these need to be sorted by TLD,
|then SLD,
|then 3LD within the Pull Request.

**Review by doitindonesia — COMMENTED** [review]
alphabetically order the sorting of the entries (Done)

**Comment by kenjiew** [comment]
Sekedar koreksi @doitindonesia 
sorry @dnsguru , I use Indonesian

Meskipun ini agak telat, tapi saya cuma mau menjelaskan dengan bahasa yang agak simple, sebenarnya @dnsguru ingin menjelaskan ke @doitindonesia untuk mengurutkan dari alfabet a-z (setahu saya) dan mengurutkan urutan dari yang paling pendek hingga panjang (misal yang pendek ada contoh.com yang panjang itu contoh.co.id dan lain sebagainya) jadi urutan yang benar adalah

a.com
b.org
c.id
d.co.id
e.co.id

jadi intinya harus urut antara a-z, 2LD dan 3LD

meskipun ini sudah usang, tapi saya sedang ada waktu luang, hehe, jadi saya menyempatkan untuk membalas ini

**Pull request closed without merging**